Feature3 - PR for including delete-a-user-Kusuma.md for assignment 6.2

KusumaKrish15 opened the pull request:
Created a new markdown file in the docs\tutorials directory about the DELETE /users/{userId} operation and created the content as described in assignment 6.2.

rbwatson commented on docs/tutorials/delete-a-user-Kusuma.md:
```diff
@@ -0,0 +1,46 @@
+            "first_name": "Johnson",
+            "email": "joe.johnson@example.com"
+        }
+4. In the Postman app, choose **Send** to make the request.
```
Add a closing code fence before this line.

rbwatson commented on docs/tutorials/delete-a-user-Kusuma.md:
Make sure your Vale and MardownLint settings are correct. See **Editing tools to improve your writing** in https://canvas.uw.edu/courses/1799216/discussion_topics/9729413

| Line | Column | Code | Message |
|------|--------|------|---------|
| 1 | 1 | `MD022` | MD022/blanks-around-headings: Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below] |
| 1 | 3 | `Google.Headings` | 'Tutorial: Delete a User' should use sentence-style capitalization. |
| 4 | 1 | `MD022` | MD022/blanks-around-headings: Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below] |
| 4 | 4 | `Google.Headings` | 'Before You Start' should use sentence-style capitalization. |
| 7 | 1 | `MD022` | MD022/blanks-around-headings: Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below] |
| 7 | 4 | `Google.Headings` | 'Delete a User' should use sentence-style capitalization. |
| 10 | 1 | `MD001` | MD001/heading-increment: Heading levels should only increment by one level at a time [Expected: h3; Actual: h4] |
| 10 | 22 | `MD026` | MD026/no-trailing-punctuation: Trailing punctuation in heading [Punctuation: ':'] |
| 13 | 1 | `MD031` | MD031/blanks-around-fences: Fenced code blocks should be surrounded by blank lines |
| 16 | 1 | `MD031` | MD031/blanks-around-fences: Fenced code blocks should be surrounded by blank lines |
| 21 | 45 | `Google.Parens` | Use parentheses judiciously. |
| 23 | 26 | `Google.WordList` | Use 'app' instead of 'application'. |
| 33 | 1 | `MD031` | MD031/blanks-around-fences: Fenced code blocks should be surrounded by blank lines |

rbwatson commented on docs/tutorials/delete-a-user-Kusuma.md:
````diff
@@ -0,0 +1,46 @@
+    }
+    ```
+  
+After doing this tutorial in Postman, you might like to repeat it in your favorite programming language. To do this, adapt the values from the tutorial to the properties and arguments that the language uses to make REST API calls.
````
```suggestion
After doing this tutorial in Postman, you might like to repeat it in your favorite programming language. To do this, adapt the values from the tutorial to the properties and arguments that the language uses to make REST API calls.

```

rbwatson reviewed (changes requested):
Please review and correct the text errors and warnings.
Be sure to add a link to this page from an existing page.

When you've made all the updates, clean up the branch as described in the articles linked in the recent announcements.

KusumaKrish15 commented:
I've incorporated the feedback and requested re-review. Please review. Thank you!

rbwatson reviewed (changes requested):
Thanks for keeping the feature branch clean!

The format of the new topic is good but the text still needs revision as described in the comments.
Also, you need to add a link to your new topic in `/docs/tutorials.md` so we can open it.

Thanks!

KusumaKrish15 commented:
Fixed the content issues based on your feedback. Added a link to the Delete a user topic from `index.md`. Re-requested review. Thanks for your patience!
